iceberg/iceberg-handler/src/test/queries/positive/iceberg_merge_files.q

```diff
@@ -0,0 +1,97 @@
+--! qt:dataset:src
+set hive.mapred.mode=nonstrict;
+set hive.explain.user=false;
+set hive.merge.mapredfiles=true;
```

Comment:
isn't that legacy config?

Reply:
Yeah but this is still being used in the tests.